app/views/works/_list_works.html.erb

```diff
@@ -0,0 +1,29 @@
+
+
+<section class="media-table">
```

Comment:
This is a great idea, but does it have to be books?  Maybe you can pass in the parameters `category` and `works`, and You can print the category in the h4 here and use `works` in the loop.